air/src/trace/main_trace.rs

```diff
@@ -92,19 +92,9 @@ impl MainTrace {
         self.addr(i) != self.addr(i + 1)
     }
 
-    /// First decoder helper register at row i.
-    pub fn helper_0(&self, i: usize) -> Felt {
-        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET)[i]
-    }
-
-    /// Second decoder helper register at row i.
-    pub fn helper_1(&self, i: usize) -> Felt {
-        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET + 1)[i]
-    }
-
-    /// Third decoder helper register at row i.
-    pub fn helper_2(&self, i: usize) -> Felt {
-        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET + 2)[i]
+    /// j-th decoder helper register at `row`.
+    pub fn helper(&self, j: usize, i: usize) -> Felt {
+        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET + j)[i]
```

Comment:
nit: the comment uses `row` while the parameter is named `i` - I think we should use `row` for the parameter name as well.

Also, if we rename `i` to `row`, I would rename `j` to `i`.

Reply:
Makes sense, I have updated `(i, j)` to `(row, i)`